test/unittest/tests/test_acquire_release_macros.cpp

```diff
@@ -72,8 +72,8 @@ TEST_RELEASE_TYPE(uint8_t, 1);
  */
 TEST_CASE("Demonstrate hash define int size workaround", "[acqrel]")
 {
-    /* If we don't cast this value the test won't compile. */
-    #define TEST_VALUE ((int8_t)6)
+/* If we don't cast this value the test won't compile. */
```

Comment:
Interesting that it stripped this whitespace, why did the change you made result in that?

Reply:
`s_clang_format` made this change before WT-12605 as well, it just didn't register it as an s_all failure. It's one of the s_all failures I saw on the head of develop per the comment on Jira.

Reply:
Is this related to s_all or it's tuning of clang_format?

Reply:
This indentation change is clang_format working correctly, but when s_clang_format makes a change it returns an error code that s-all didn't detect. 
In the PR where this line was added clang_format would have corrected the indentation, but the `s-all` task incorrectly reported success during PR testing.

Reply:
@ajmorton Did you use the [-E](https://github.com/wiredtiger/wiredtiger/blob/cbbfe891b74f45f6a47359a72256c1a1343c434d/dist/s_all#L68) flag?

Reply:
It didn't detect the error previously but now it does so. That's why I had to include this fix into this PR - otherwise s-all was failing.

Remember that to get a correct error status, you need to use the [-E](https://github.com/wiredtiger/wiredtiger/blob/cbbfe891b74f45f6a47359a72256c1a1343c434d/dist/s_all#L68) flag if you run s_all locally.